src/main/scala/nl/knaw/dans/lib/logging/servlet/masked/Masker.scala

```diff
@@ -46,8 +46,8 @@ private[masked] object Masker {
     remoteAddress.replaceAll("([0-9]+[.]){3}", "**.**.**.")
   }
 
-  def formatCookieHeader(headerName: String): HeaderMapEntry => HeaderMapEntry = {
-    formatTuple(_.toLowerCase == headerName)(formatCookie)
+  def formatCookieHeader(headerName: String)(formatter: String => String): HeaderMapEntry => HeaderMapEntry = {
```

Comment:
Shouldn't the other methods also be curried, or if no use case (likely, I think) made private?